applications/tari_console_wallet/src/main.rs

```diff
@@ -176,3 +160,23 @@ fn main_inner() -> Result<(), ExitCodes> {
 
     result
 }
+
+fn get_master_key(boot_mode: WalletBoot, bootstrap: &ConfigBootstrap) -> Result<Option<PrivateKey>, ExitCodes> {
+    if matches!(boot_mode, WalletBoot::Recovery) {
+        let private_key = if bootstrap.seed_words.is_some() {
```

Comment:
Minor: I know this was from a previous PR, but when I see `is_some()` I think it should be `if let Some` instead.